app/v1/controllers/requestParsers/validators/DeletePensionsReliefsValidator.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.controllers.requestParsers.validators
+
+import v1.controllers.requestParsers.validators.validations.{NinoValidation, TaxYearValidation}
+import v1.models.errors.MtdError
+import v1.models.request.deletePensionsReliefs.DeletePensionsReliefsRawData
+
+class DeletePensionsReliefsValidator extends Validator[DeletePensionsReliefsRawData] {
+
+  private val validationSet = List(parameterFormatValidation)
+
+  private def parameterFormatValidation: DeletePensionsReliefsRawData => List[List[MtdError]] = data => {
+    List(
+      NinoValidation.validate(data.nino),
+      TaxYearValidation.validate(data.taxYear)
```

Comment:
The spec includes the `RULE_TAX_YEAR_NOT_SUPPORTED` error but currently the taxYearValidation doesn't include that error.

(Though keep in mind the investments, other and foreign endpoints don't include the error yet)